mlos_bench/mlos_bench/config/schemas/config_schemas.py

```diff
@@ -56,6 +56,8 @@ def _load_schemas(cls) -> None:
                     schema = json.load(schema_file)
                     cls._SCHEMA_STORE[file_path] = schema
                     # Let the schema be referenced by its id as well.
+                    assert "$id" in schema
+                    assert schema["$id"] not in cls._SCHEMA_STORE
```

Comment:
Make sure we don't overwrite a schema by id - that's an error in schema development.